src/components/chip-set/chip-set.tsx

```diff
@@ -572,9 +578,11 @@ export class ChipSet {
         return (
             <i
```

Comment:
I think this should be a link. Then there should be no need for the `tabindex="0"`, and it ought to be possible to trigger it by tabbing to it and pressing \<enter\>.

Alternatively, we might want to use a limel-icon-button, but I'm guessing that might be a little too bloated for use here?

Reply:
have you tried to see whether having it as a link would work? Otherwise I'lll test tomorrow ;)

Reply:
@adrianschmidt this didn't work unfortunately. :( maybe for a link to work, we need a URL? donno